### PR TITLE
added manual  update guide for ubuntu

### PR DESCRIPTION
**Pull request by omar-saadoun**
added manual  update guide for ubuntu, should be the base for adding more node update documentation

### CODE REVIEW

File: docs/update.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+* [Manual update](#manual-update);
+
+## Manual update
+
+### Ubuntu 18.04
+
+Be sure to have a backup of your files, don't remove your node database.
+
+Download the latest version of æternity node (Iris is 6.0.0)
```
Comment (uwiger) [inline docs/update.md:13]: The latest version of Iris is `6.1.0`, but perhaps we shouldn't hard-code the latest version here. "(Iris is 6.x.y)", perhaps, or simply "Download the latest version of the aeternity node".

File: docs/update.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+```
+/path/to/your/node/bin/aeternity daemon
+```
+
+Now check that your node is up and running by :
+
+
+```
+curl localhost:3013/v2/blocks/top
```
Comment (velzevur) [inline docs/update.md:41]: What if the port of the external HTTP interface is being set to a different value?

**Comment by velzevur** [comment]
@omar-saadoun please add a line regarding updates in [/docs/README.md](https://github.com/aeternity/aeternity/blob/a5d89cbce00965a08a393e1ae9db99703bf37d24/docs/README.md).

**Review by zxq9 — APPROVED** [review]
Yay!

**Review by seanhinde — APPROVED** [review]
Nice update

**Pull request merged**